platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/crypto/CryptoArgsProvider.java

```diff
@@ -53,9 +54,14 @@ static Stream<Arguments> basicTestArgs() throws Exception {
         final Map<NodeId, KeysAndCerts> genC = CryptoStatic.generateKeysAndCerts(genAB);
         System.out.println(
                 "Key generating took " + Duration.between(start, Instant.now()).toMillis());
+        start = Instant.now();
+        final AddressBookAndCerts addressBookAndCerts2 = createAddressBookLoadSigKeysCreateAgrKeys(NUMBER_OF_ADDRESSES);
+        System.out.println("Agreement key generating took "
+                + Duration.between(start, Instant.now()).toMillis());
```

Comment:
This method doesn't need to be printing (I know this is legacy behavior, but might be a good time to clean it up since we are touching it). In general, no need to print/log unless a test doesn't pass.

Reply:
removed all print statements from method.